cannot find crate

Rust compiler:

rustup show
Default host: x86_64-unknown-linux-gnu

stable-x86_64-unknown-linux-gnu (default)
rustc 1.35.0 (3c235d560 2019-05-20)


Project: https://github.com/svenschmidt75/Rust, commit 78b9a8fd4740bbf8630cc0ade48346a5387f2075, authored 6/2/2019 @ 8:58 AM

File model.rs, line 139,
```
...
                    mb.a[0] = training_sample.input_activations.clone();
                    self.feedforward(mb);
                    self.backprop(mb, cost_function, known_classification);
                }
                let (dws, dbs) = self.calculate_derivatives(&mbs[..], lambda);
                self.apply_momentum(eta, rho, dws, dbs);
                self.update_network();
            }
...
```

Replace `eta` with `meta` and compile. The error refers to a missing crate which I found odd:
```
   Compiling ann v0.1.0 (/home/svenschmidt75/Develop/Rust/NeuralNetwork/lib/ann)
error[E0463]: can't find crate for `meta`
   --> lib/ann/src/ann/model.rs:139:37
    |
139 |                 self.apply_momentum(meta, rho, dws, dbs);
    |                                     ^^^^ can't find crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `ann`.
warning: build failed, waiting for other jobs to finish...
error[E0463]: can't find crate for `meta`
   --> lib/ann/src/ann/model.rs:139:37
    |
139 |                 self.apply_momentum(meta, rho, dws, dbs);
    |                                     ^^^^ can't find crate

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `ann`.

To learn more, run the command again with --verbose.

Process finished with exit code 101
```

<!-- TRIAGEBOT_START -->

<!-- TRIAGEBOT_ASSIGN_START -->

<!-- TRIAGEBOT_ASSIGN_END -->
<!-- TRIAGEBOT_END -->

Reproduction:

    let xyz = meta;

(ie. this happens to any occurrence of a nonexistent local `meta`)

Expected outcome would be ``error[E0425]: cannot find value `meta` in this scope``

Current output:

```
error[E0425]: cannot find value `meta` in this scope
 --> src/main.rs:2:15
  |
2 |     let xyz = meta;
  |               ^^^^ not found in this scope
```

--- a/tests/ui/reserved/meta-is-not-reserved.rs
+++ b/tests/ui/reserved/meta-is-not-reserved.rs
@@ -1,0 +1,7 @@
+// Regression test for <https://github.com/rust-lang/rust/issues/61463>
+// A test for the issue where the variable meta is mistakenly treated as a reserved keyword.
+
+fn main() {
+    let xyz = meta;
+    //~^ ERROR cannot find value `meta` in this scope [E0425]
+}

--- a/tests/ui/reserved/meta-is-not-reserved.stderr
+++ b/tests/ui/reserved/meta-is-not-reserved.stderr
@@ -1,0 +1,9 @@
+error[E0425]: cannot find value `meta` in this scope
+  --> $DIR/meta-is-not-reserved.rs:5:15
+   |
+LL |     let xyz = meta;
+   |               ^^^^ not found in this scope
+
+error: aborting due to 1 previous error
+
+For more information about this error, try `rustc --explain E0425`.

